### ISSUE TITLE
[BUG] <title>Search function not working

### ISSUE THREAD

**Issue by Vextryyn** [linuxserver/docker-sonarr#302]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Version 4.0.10 
does not search correctly or apply tags.
Upgraded through Portainer 2.21.4

When using interactive search it searches first for the season then searches each individual episode in the series, logs show that it is searching and not interactive searching. When the interactive search finally loads you are unable to add any of the items, it will say it does, but nothing happens, this error seems to show up when doing that error code="500" description="Memory stream is not expandable."(I have 16GB of ram in the machine running it and also tried increasing to 32

Downgrading to 4.0.9 resolved the issue

### Steps To Reproduce

Upgrade 4.0.9 to 4.0.10
Search or interactive search

### Environment

```markdown
- OS: Ubuntu 24.04.1
- How docker service was installed: Portainer, under containers select existing Sonarr install, edit/duplicate, verify repo states latest and Deploy
```

### CPU architecture

x86-64

### Docker creation

```bash
Imported through new container creation in Portainer under containers select existing Sonarr install, edit/duplicate, verify repo states latest and Deploy 

ENV:
PGID=0
PUID=0
PATH=/lsiopy/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
PS1=$(whoami)@$(hostname):$(pwd)\$
HOME=/root
TERM=xterm
S6_CMD_WAIT_FOR_SERVICES_MAXTIME=0
S6_VERBOSITY=1
S6_STAGE2_HOOK=/docker-mods
VIRTUAL_ENV=/lsiopy
LSIO_FIRST_PARTY=true
XDG_CONFIG_HOME=/config/xdg
SONARR_CHANNEL=v4-stable
SONARR_BRANCH=main
COMPlus_EnableDiagnostics=0
TMPDIR=/run/sonarr-temp
```

### Container logs

```bash
[Info] DownloadService: Report sent to qBittorrent. Indexer Nyaa.si (Prowlarr). [New-raws] Ore dake Level Up na Ken - 02 [1080p] [KOR DUB].mkv 
[Info] DownloadService: Report sent to qBittorrent. Indexer Nyaa.si (Prowlarr). [New-raws] Ore dake Level Up na Ken - 03 [1080p] [KOR DUB].mkv 
[Info] DownloadService: Report sent to qBittorrent. Indexer Nyaa.si (Prowlarr). [New-raws] Ore dake Level Up na Ken - 04 [1080p] [KOR DUB].mkv 
[Info] DownloadService: Report sent to qBittorrent. Indexer Nyaa.si (Prowlarr). [New-raws] Ore dake Level Up na Ken - 08 [1080p] [KOR DUB].mkv 
[Info] SeriesSearchService: Series search completed. 5 reports downloaded. 
[Warn] HttpClient: HTTP Error - Res: HTTP/1.1 [GET] http://localhost:9696/3/download?apikey=3f0df7905b7943bda7f778b6d4cc5361&link=MCtlN3hQN2c1SEdtbHhnWndLRGk0bFNMVzMzdnBBOS8vR2RXbG9PTGNmNGVGUDdvUlo1RU0ySEhJWUVyV05Tb3JnN2VsZ3N0TE12MDNGaGU0NzFqZGI1d1g3cko5MHdqWk1yN1RrSzdxeVZZT0t3Q1JhbDBNL3VQMm9RNDhSeEt5T1R0czluS09PL2NaK1FXS1VGek5IbFNueU5zRWc2TDhMUkR4a3ZzaDYwPQ&file=The+Witch+and+the+Beast+%5BSeason+1%5D+%5BWEB+1080p+HEVC+AAC%5D+%5BDual+Audio+EngSubs%5D+(Batch): 500.InternalServerError (106 bytes)
<?xml version="1.0" encoding="UTF-8"?>
<error code="500" description="Memory stream is not expandable." /> 
[Info] QBittorrent: Request for qBittorrent failed with exception 'HTTP request failed: [500:InternalServerError] [GET] at [http://localhost:9696/3/download?apikey=3f0df7905b7943bda7f778b6d4cc5361&link=MCtlN3hQN2c1SEdtbHhnWndLRGk0bFNMVzMzdnBBOS8vR2RXbG9PTGNmNGVGUDdvUlo1RU0ySEhJWUVyV05Tb3JnN2VsZ3N0TE12MDNGaGU0NzFqZGI1d1g3cko5MHdqWk1yN1RrSzdxeVZZT0t3Q1JhbDBNL3VQMm9RNDhSeEt5T1R0czluS09PL2NaK1FXS1VGek5IbFNueU5zRWc2TDhMUkR4a3ZzaDYwPQ&file=The+Witch+and+the+Beast+%5BSeason+1%5D+%5BWEB+1080p+HEVC+AAC%5D+%5BDual+Audio+EngSubs%5D+(Batch)]'. Retrying in 2.4300412s. 
[v4.0.10.2544] NzbDrone.Common.Http.HttpException: HTTP request failed: [500:InternalServerError] [GET] at [http://localhost:9696/3/download?apikey=3f0df7905b7943bda7f778b6d4cc5361&link=MCtlN3hQN2c1SEdtbHhnWndLRGk0bFNMVzMzdnBBOS8vR2RXbG9PTGNmNGVGUDdvUlo1RU0ySEhJWUVyV05Tb3JnN2VsZ3N0TE12MDNGaGU0NzFqZGI1d1g3cko5MHdqWk1yN1RrSzdxeVZZT0t3Q1JhbDBNL3VQMm9RNDhSeEt5T1R0czluS09PL2NaK1FXS1VGek5IbFNueU5zRWc2TDhMUkR4a3ZzaDYwPQ&file=The+Witch+and+the+Beast+%5BSeason+1%5D+%5BWEB+1080p+HEVC+AAC%5D+%5BDual+Audio+EngSubs%5D+(Batch)]
   at NzbDrone.Common.Http.HttpClient.ExecuteAsync(HttpRequest request) in ./Sonarr.Common/Http/HttpClient.cs:line 119
   at NzbDrone.Core.Download.TorrentClientBase`1.<>c.<<DownloadFromWebUrl>b__11_0>d.MoveNext() in ./Sonarr.Core/Download/TorrentClientBase.cs:line 143
--- End of stack trace from previous location ---
   at Polly.ResiliencePipeline.<>c__9`2.<<ExecuteAsync>b__9_0>d.MoveNext()
<?xml version="1.0" encoding="UTF-8"?>
<error code="500" description="Memory stream is not expandable." />
[Info] DownloadDecisionMaker: Processing 36 releases 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01E06 (06)]. 3 active indexers 
[Info] ReleaseSearchService: Searching indexers for [Solo Leveling : S01]. 3 active indexers 
[Warn] HttpClient: HTTP Error - Res: HTTP/1.1 [GET] http://localhost:9696/3/download?apikey=3f0df7905b7943bda7f778b6d4cc5361&link=MCtlN3hQN2c1SEdtbHhnWndLRGk0bFNMVzMzdnBBOS8vR2RXbG9PTGNmNGVGUDdvUlo1RU0ySEhJWUVyV05Tb3JnN2VsZ3N0TE12MDNGaGU0NzFqZGI1d1g3cko5MHdqWk1yN1RrSzdxeVZZT0t3Q1JhbDBNL3VQMm9RNDhSeEt5T1R0czluS09PL2NaK1FXS1VGek5IbFNueU5zRWc2TDhMUkR4a3ZzaDYwPQ&file=The+Witch+and+the+Beast+%5BSeason+1%5D+%5BWEB+1080p+HEVC+AAC%5D+%5BDual+Audio+EngSubs%5D+(Batch): 500.InternalServerError (106 bytes)
<?xml version="1.0" encoding="UTF-8"?>
<error code="500" description="Memory stream is not expandable." /> 
[Info] QBittorrent: Request for qBittorrent failed with exception 'HTTP request failed: [500:InternalServerError] [GET] at [http://localhost:9696/3/download?apikey=3f0df7905b7943bda7f778b6d4cc5361&link=MCtlN3hQN2c1SEdtbHhnWndLRGk0bFNMVzMzdnBBOS8vR2RXbG9PTGNmNGVGUDdvUlo1RU0ySEhJWUVyV05Tb3JnN2VsZ3N0TE12MDNGaGU0NzFqZGI1d1g3cko5MHdqWk1yN1RrSzdxeVZZT0t3Q1JhbDBNL3VQMm9RNDhSeEt5T1R0czluS09PL2NaK1FXS1VGek5IbFNueU5zRWc2TDhMUkR4a3ZzaDYwPQ&file=The+Witch+and+the+Beast+%5BSeason+1%5D+%5BWEB+1080p+HEVC+AAC%5D+%5BDual+Audio+EngSubs%5D+(Batch)]'. Retrying in 2.9999552s. 
[v4.0.10.2544] NzbDrone.Common.Http.HttpException: HTTP request failed: [500:InternalServerError] [GET] at [http://localhost:9696/3/download?apikey=3f0df7905b7943bda7f778b6d4cc5361&link=MCtlN3hQN2c1SEdtbHhnWndLRGk0bFNMVzMzdnBBOS8vR2RXbG9PTGNmNGVGUDdvUlo1RU0ySEhJWUVyV05Tb3JnN2VsZ3N0TE12MDNGaGU0NzFqZGI1d1g3cko5MHdqWk1yN1RrSzdxeVZZT0t3Q1JhbDBNL3VQMm9RNDhSeEt5T1R0czluS09PL2NaK1FXS1VGek5IbFNueU5zRWc2TDhMUkR4a3ZzaDYwPQ&file=The+Witch+and+the+Beast+%5BSeason+1%5D+%5BWEB+1080p+HEVC+AAC%5D+%5BDual+Audio+EngSubs%5D+(Batch)]
   at NzbDrone.Common.Http.HttpClient.ExecuteAsync(HttpRequest request) in ./Sonarr.Common/Http/HttpClient.cs:line 119
   at NzbDrone.Core.Download.TorrentClientBase`1.<>c.<<DownloadFromWebUrl>b__11_0>d.MoveNext() in ./Sonarr.Core/Download/TorrentClientBase.cs:line 143
--- End of stack trace from previous location ---
   at Polly.ResiliencePipeline.<>c__9`2.<<ExecuteAsync>b__9_0>d.MoveNext()
<?xml version="1.0" encoding="UTF-8"?>
<error code="500" description="Memory stream is not expandable." />
[Warn] HttpClient: HTTP Error - Res: HTTP/1.1 [GET] http://localhost:9696/3/download?apikey=3f0df7905b7943bda7f778b6d4cc5361&link=MCtlN3hQN2c1SEdtbHhnWndLRGk0bFNMVzMzdnBBOS8vR2RXbG9PTGNmNGVGUDdvUlo1RU0ySEhJWUVyV05Tb3JnN2VsZ3N0TE12MDNGaGU0NzFqZGI1d1g3cko5MHdqWk1yN1RrSzdxeVZZT0t3Q1JhbDBNL3VQMm9RNDhSeEt5T1R0czluS09PL2NaK1FXS1VGek5IbFNueU5zRWc2TDhMUkR4a3ZzaDYwPQ&file=The+Witch+and+the+Beast+%5BSeason+1%5D+%5BWEB+1080p+HEVC+AAC%5D+%5BDual+Audio+EngSubs%5D+(Batch): 500.InternalServerError (106 bytes)
<?xml version="1.0" encoding="UTF-8"?>
<error code="500" description="Memory stream is not expandable." /> 
[Error] QBittorrent: Downloading torrent file for episode 'The Witch and the Beast [Season 1] [WEB 1080p HEVC AAC] [Dual Audio EngSubs] (Batch)' failed (http://localhost:9696/3/download?apikey=3f0df7905b7943bda7f778b6d4cc5361&link=MCtlN3hQN2c1SEdtbHhnWndLRGk0bFNMVzMzdnBBOS8vR2RXbG9PTGNmNGVGUDdvUlo1RU0ySEhJWUVyV05Tb3JnN2VsZ3N0TE12MDNGaGU0NzFqZGI1d1g3cko5MHdqWk1yN1RrSzdxeVZZT0t3Q1JhbDBNL3VQMm9RNDhSeEt5T1R0czluS09PL2NaK1FXS1VGek5IbFNueU5zRWc2TDhMUkR4a3ZzaDYwPQ&file=The+Witch+and+the+Beast+%5BSeason+1%5D+%5BWEB+1080p+HEVC+AAC%5D+%5BDual+Audio+EngSubs%5D+(Batch)) 
[v4.0.10.2544] NzbDrone.Common.Http.HttpException: HTTP request failed: [500:InternalServerError] [GET] at [http://localhost:9696/3/download?apikey=3f0df7905b7943bda7f778b6d4cc5361&link=MCtlN3hQN2c1SEdtbHhnWndLRGk0bFNMVzMzdnBBOS8vR2RXbG9PTGNmNGVGUDdvUlo1RU0ySEhJWUVyV05Tb3JnN2VsZ3N0TE12MDNGaGU0NzFqZGI1d1g3cko5MHdqWk1yN1RrSzdxeVZZT0t3Q1JhbDBNL3VQMm9RNDhSeEt5T1R0czluS09PL2NaK1FXS1VGek5IbFNueU5zRWc2TDhMUkR4a3ZzaDYwPQ&file=The+Witch+and+the+Beast+%5BSeason+1%5D+%5BWEB+1080p+HEVC+AAC%5D+%5BDual+Audio+EngSubs%5D+(Batch)]
   at NzbDrone.Common.Http.HttpClient.ExecuteAsync(HttpRequest request) in ./Sonarr.Common/Http/HttpClient.cs:line 119
   at NzbDrone.Core.Download.TorrentClientBase`1.<>c.<<DownloadFromWebUrl>b__11_0>d.MoveNext() in ./Sonarr.Core/Download/TorrentClientBase.cs:line 143
--- End of stack trace from previous location ---
   at Polly.ResiliencePipeline.<>c__9`2.<<ExecuteAsync>b__9_0>d.MoveNext()
--- End of stack trace from previous location ---
   at Polly.Outcome`1.GetResultOrRethrow()
   at Polly.ResiliencePipeline.ExecuteAsync[TResult,TState](Func`3 callback, TState state, CancellationToken cancellationToken)
   at NzbDrone.Core.Download.TorrentClientBase`1.DownloadFromWebUrl(RemoteEpisode remoteEpisode, IIndexer indexer, String torrentUrl) in ./Sonarr.Core/Download/TorrentClientBase.cs:line 215
<?xml version="1.0" encoding="UTF-8"?>
<error code="500" description="Memory stream is not expandable." />
[Error] ReleaseController: Downloading torrent failed 
[v4.0.10.2544] NzbDrone.Core.Exceptions.ReleaseDownloadException: Downloading torrent failed
 ---> NzbDrone.Common.Http.HttpException: HTTP request failed: [500:InternalServerError] [GET] at [http://localhost:9696/3/download?apikey=3f0df7905b7943bda7f778b6d4cc5361&link=MCtlN3hQN2c1SEdtbHhnWndLRGk0bFNMVzMzdnBBOS8vR2RXbG9PTGNmNGVGUDdvUlo1RU0ySEhJWUVyV05Tb3JnN2VsZ3N0TE12MDNGaGU0NzFqZGI1d1g3cko5MHdqWk1yN1RrSzdxeVZZT0t3Q1JhbDBNL3VQMm9RNDhSeEt5T1R0czluS09PL2NaK1FXS1VGek5IbFNueU5zRWc2TDhMUkR4a3ZzaDYwPQ&file=The+Witch+and+the+Beast+%5BSeason+1%5D+%5BWEB+1080p+HEVC+AAC%5D+%5BDual+Audio+EngSubs%5D+(Batch)]
   at NzbDrone.Common.Http.HttpClient.ExecuteAsync(HttpRequest request) in ./Sonarr.Common/Http/HttpClient.cs:line 119
   at NzbDrone.Core.Download.TorrentClientBase`1.<>c.<<DownloadFromWebUrl>b__11_0>d.MoveNext() in ./Sonarr.Core/Download/TorrentClientBase.cs:line 143
--- End of stack trace from previous location ---
   at Polly.ResiliencePipeline.<>c__9`2.<<ExecuteAsync>b__9_0>d.MoveNext()
--- End of stack trace from previous location ---
   at Polly.Outcome`1.GetResultOrRethrow()
   at Polly.ResiliencePipeline.ExecuteAsync[TResult,TState](Func`3 callback, TState state, CancellationToken cancellationToken)
   at NzbDrone.Core.Download.TorrentClientBase`1.DownloadFromWebUrl(RemoteEpisode remoteEpisode, IIndexer indexer, String torrentUrl) in ./Sonarr.Core/Download/TorrentClientBase.cs:line 215
<?xml version="1.0" encoding="UTF-8"?>
<error code="500" description="Memory stream is not expandable." />
   --- End of inner exception stack trace ---
   at NzbDrone.Core.Download.TorrentClientBase`1.DownloadFromWebUrl(RemoteEpisode remoteEpisode, IIndexer indexer, String torrentUrl) in ./Sonarr.Core/Download/TorrentClientBase.cs:line 215
   at NzbDrone.Core.Download.TorrentClientBase`1.Download(RemoteEpisode remoteEpisode, IIndexer indexer) in ./Sonarr.Core/Download/TorrentClientBase.cs:line 129
   at NzbDrone.Core.Download.DownloadService.DownloadReport(RemoteEpisode remoteEpisode, IDownloadClient downloadClient) in ./Sonarr.Core/Download/DownloadService.cs:line 143
   at NzbDrone.Core.Download.DownloadService.DownloadReport(RemoteEpisode remoteEpisode, Nullable`1 downloadClientId) in ./Sonarr.Core/Download/DownloadService.cs:line 64
   at Sonarr.Api.V3.Indexers.ReleaseController.DownloadRelease(ReleaseResource release) in ./Sonarr.Api.V3/Indexers/ReleaseController.cs:line 160
[Info] DownloadDecisionMaker: Processing 40 releases 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01E07 (07)]. 3 active indexers 
[Info] DownloadDecisionMaker: Processing 181 releases 
[Info] ReleaseSearchService: Searching indexers for [Solo Leveling : S01E01 (01)]. 3 active indexers 
[Info] DownloadDecisionMaker: Processing 85 releases 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01E08 (08)]. 3 active indexers 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01]. 3 active indexers 
[Info] DownloadDecisionMaker: Processing 108 releases 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01E01 (01)]. 3 active indexers 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01]. 3 active indexers 
[Info] DownloadDecisionMaker: Processing 44 releases 
[Info] DownloadDecisionMaker: Processing 219 releases 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01E09 (09)]. 3 active indexers 
[Info] ReleaseSearchService: Searching indexers for [Solo Leveling : S01E02 (02)]. 3 active indexers 
[Info] DownloadDecisionMaker: Processing 108 releases 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01E01 (01)]. 3 active indexers 
[Info] DownloadDecisionMaker: Processing 148 releases 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01E02 (02)]. 3 active indexers 
[Info] DownloadDecisionMaker: Processing 38 releases 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01E10 (10)]. 3 active indexers 
[Info] DownloadDecisionMaker: Processing 148 releases 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01E02 (02)]. 3 active indexers 
[Info] DownloadDecisionMaker: Processing 124 releases 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01E03 (03)]. 3 active indexers 
[Info] DownloadDecisionMaker: Processing 206 releases 
[Info] ReleaseSearchService: Searching indexers for [Solo Leveling : S01E03 (03)]. 3 active indexers 
[Info] DownloadDecisionMaker: Processing 120 releases 
[Info] ReleaseSearchService: Searching indexers for [The Witch and the Beast : S01E11 (11)]. 3 active indexers
```

**Comment by Roxedus** [comment]
Not a container issue, seek support from sonarr directly.

**Comment by Vextryyn** [comment]
> Not a container issue, seek support from sonarr directly.

well, my buddy uses unraid and has verified that the binhex version for arch works perfectly without any of these issues. I also spun up my windows install and the windows version works perfectly as well. for the giggles, i installed the Hotio version, and once again there are no issues. The issue is with your container not sonarr

**Comment by Roxedus** [comment]
The issue is due to a badly crafted torrent file, which you can only replicate with the same indexers/custom formats and blocklists.

**Comment by Vextryyn** [comment]
> The issue is due to a badly crafted torrent file, which you can only replicate with the same indexers/custom formats and blocklists.
Restoring the exact same backup on the Hotio install does not produce this issue. Searching prowlarr directly produces the correct results in a timely manner.

**Comment by Roxedus** [comment]
Don't know what else to tell you. That error is caused by a malformed torrent file (prowlarr doesn't necessarily check this)

**Comment by Vextryyn** [comment]
> Don't know what else to tell you. That error is caused by a malformed torrent file (prowlarr doesn't necessarily check this)

so literally checking the same show on the Hotio, Windows and Binhex installs, all running from the exact same backup, somehow means there is a malformed torrent file that is causing the issue only on your container? Really starting to make sense why everyone tells me to avoid the linuxserver containers.

**Comment by Roxedus** [comment]
Get us something we can reproduce this issue with. I can not see how this is a container issue, without having concrete path to troubleshoot. I researched online, and everything points towards a malformed torrent file.

Have you verified from the logs, that the other installs also gets the same release from the same indexer?

**Comment by Vextryyn** [comment]
> Get us something we can reproduce this issue with. I can not see how this is a container issue, without having concrete path to troubleshoot. I researched online, and everything points towards a malformed torrent file.
> 
> Have you verified from the logs, that the other installs also gets the same release from the same indexer?

The search function on other installs applies my tags correctly doing the automatic search and ACTUALLY DOWNLOADS A TORRENT. Interactive search pulls the results instantly and i dont have a log filled with a search for each individual episode before the results appear and guess what? WHEN I PRESS DOWNLOAD IT DOWNLOADS. 

what else could i possibly provide that isnt already in my ticket? my install is as vanilla as you can get, its ported through Glutun using prowlarr 1337x, nyaa, tpb +Flaresolvrr

**Comment by Roxedus** [comment]
Thats far from a vanilla setup. But post tracelogs here if you don't want to report report this upstream where it belongs.

**Comment by j0nnymoe** [comment]
As a side note, the only other thing I can think of is portainer potentially messing with the container creation. If you're willing, could you try creating the container with compose?

**Comment by Vextryyn** [comment]
> As a side note, the only other thing I can think of is portainer potentially messing with the container creation. If you're willing, could you try creating the container with compose?

That seems to be a fix, none of the issues I've listed occur when I install through compose and restore the same backup, but when i install as a container and restore the exact same backup the issues are still the same. going through the settings everything is identical post install.

**Comment by aptalca** [comment]
> Really starting to make sense why everyone tells me to avoid the linuxserver containers.

Now take that back 😜

**Comment by Vextryyn** [comment]
> The issue can be reproduced easily by grabbing the same exact `The Witch and the Beast` release from itorrents.
> 
> This is not a LSIO nor a Prowlarr issue, just corrupted torrent files generated by a 3rd-party service. I pushed a change to Prowlarr to report it's an `Invalid torrent file contents` to prevent any possible confusion.

everything marked [Info] is an interactive search not an automatic search. all of the errors before [Info] are from the automatic search. 

That said, im gonna move this issue over to Sonarr. After reviewing my interactive results for multiple different types of installs The Witch and the Beast pulls a large number of extra results that are not related, but the correct results in the previous version. Solo Leveling yields 0 search results again plenty of correct results in the previous version. Im also noticing issues with results for a few other Anime as well, once again all good prev ver. This may also be why the latest Radarr downloaded Liar Liar instead of Princess Mononoke. 

im not showing any of the same info in the logs btw, so yall seem to have all the extra logs, kudos, im just gonna stay a version behind and prolly roll back my radarr as well.

**Comment by j0nnymoe** [comment]
> > As a side note, the only other thing I can think of is portainer potentially messing with the container creation. If you're willing, could you try creating the container with compose?
> 
> That seems to be a fix, none of the issues I've listed occur when I install through compose and restore the same backup, but when i install as a container and restore the exact same backup the issues are still the same. going through the settings everything is identical post install.

The issue you're experiencing is something portainer is breaking and not the first time we've seen it with web based container management software: https://info.linuxserver.io/issues/2023-05-22-portainer/

**Comment by Vextryyn** [comment]
few things, like the link above said, I needed to setup from scratch in order to get it all working correctly again, but after messing with things for a bit, found how to make it work for others without losing all of the settings. at least once i got it all updated the interactive search now shows correctly, results load quickly, it searches for what it is supposed to, tags apply correctly in the auto search and the gui correctly shows that there was an error adding it to the client and what the error is. 

the issue that alot of you are hung up on is the auto search errors, which is not what i was concerned about, but fyi those were related to nyaa not working, passed it through flaresolvrr and that seems to have cleared up that issue. im used to things erroring right away and not working for tests if it requires flaresolverr. 

all the errors i was actually concerned about (tags applying, interactive search issues, not adding to client) persisted even after fixing nyaa.  

while it did originally seem to only affect anime, i did end up discovering that more obsucure shows were also having the issue of loading every torrent in history in the search results. 

to [j0nnymoe](https://github.com/j0nnymoe), sorry for the thumbs down, I did not realize i thumbsed down until i hopped on to share my findings today(this is why i dont use my phone for things), that was supposed to be a thumbs up cause the info in the link made enough sense to move forward. 

pretty much spent the last 2 weeks trying to replicate what was happening and how to ensure i dont lose everything when sonarr updates and im fairly confident this is the best way to ensure you dont need to start from scratch every update in portainer:

1. Before updating, in sonarr/radarr make a backup of the old ver pre update.(if you backup post update it will never work correctly) note: i had already effed this part up so i had to start from scratch which means the back up was loading corrupted settings
2. delete the container or at least stop the container
3. create a new config volume or delete the data inside the old one, the old data will break it post update(infinite manual searching for no reason, lack of useful gui errors, etc)
4. create a new container and install the new version
5. restore the backup in sonarr/radarr and it will function correctly.
